affiliate/src/productCard/Edit.jsx

```diff
@@ -41,7 +41,7 @@ const Edit = ({ attributes, setAttributes, productSearchPageUrl }) => {
     attributes.displayDescription,
     attributes.bannerText,
   ]);
-  const [showModal, setShowModal] = useState(!attributes.productGuid);
+  const [showModal, setShowModal] = useState(false);
```

Comment:
I like the behavior of the modal opening automatically upon first selecting the integration, in other words, without having to press the icon; however, I also see how this can be an issue since the same logic will open the modal whenever the editor visits a page that has an unfinished integration. Maybe we could add an attribute so that the modal pops up by default the first time the user is building the integration?

Reply:
Yeah, I must have been visiting a page that had an unfinished one without realizing it. I reverted this.